CIP-interoperable-programmable-tokens/README.md

```diff
@@ -0,0 +1,154 @@
+---
+CIP: ???
+Title: Interoperable Programmable Tokens
+Category: Plutus
```

Comment:
```suggestion
Category: Tokens
```
highlighting discussion question from https://github.com/cardano-foundation/CIPs/pull/944#pullrequestreview-2481913671:
> An essential topic for the meeting & assigning a CIP number will be to confirm this is really a `Plutus` category proposal as you've originally written, or if it would be more useful & meaningful to categorise it `Tokens` like those others. I'll leave the `Category:` tag off until we can confirm it here or at the meeting.

Reply:
Resolution from today's CIP meeting is that this can go forward with `Tokens` category - even though some parts of the proposal would benefit from optimisations possible if some changes in Plutus are made - and that @colll78 will produce a separate CIP with the recommended Plutus changes.

Reply:
fixed in https://github.com/cardano-foundation/CIPs/pull/944/commits/df276867ed686f060fc099b09bf476af287fb20d